recipes/state-machine-cat/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "state-machine-cat" %}
+{% set version = "v10.1.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_")|replace("v", "") }}
+
+source:
+  url: https://github.com/sverweij/state-machine-cat/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 386d2578731f0899c540e5020b4627cc75af4e50fb61887505aac47bd5c9dea1
```

Is it possible to use the distribution at npmjs? That one is leaner to download as it only consists of the sources strictly necessary for running  state-machine-cat (~350kb) and doesn't contain the 3.5Mb of tests and 22Mb of website/ docs.

```suggestion
  url: https://registry.npmjs.org/state-machine-cat/-/state-machine-cat-{{ version|replace("v", "") }}.tgz
  sha256: 7348eaab734296dfa9aa046ad7ce20e1d64696c8e81c73019e08e3ad22b18e9a
```

:point_up: this sha256 is  b.t.w. the one for v11.0.0

---

Not sure, but I can try that. 

Where's the sha256 for v11.0.0 documented by the way?

On the releases page, it says this, which is different from above.
> shasum of the package as published on npmjs: 1a3629148c765a814fd1a837df75b1ccfe21cb4d

---

npmjs seems to have only an md5 and a sha512 (you can check with `npm view state-machine-cat`) readily available. If conda-forge can handle a sha512 that would  be easiest.

I got the sha256 by downloading the archive and calculating  the sha256 from  it:

```sh
wget https://registry.npmjs.org/state-machine-cat/-/state-machine-cat-11.0.0.tgz
openssl sha256 state-machine-cat-11.0.0.tgz
```

---

Unfortunately `sha512` is not available yet (there's an open PR for that https://github.com/conda/conda-build/pull/4793). 

I'm using `sha256` for the time being. 

And the npm registry source works perfectly.